### DISCUSSION TITLE
Feature Request: Crash/Emergency break autorecord

### DISCUSSION THREAD

**Discussion by viskor** [gopro/labs#203] [General]
Hi,

​I have a gp11 and record ​my bike cession.
I actually​ use a remote with hindsight for recording all problème or incident on my road but i need that my gopro11 auto lauch if i crash me or if a car hit me and if Its not possible to me to start record(push button record).

​I see that it's possible to lunch vidéo at xx speed but in my case​, i need to auto record When my speed go down fast or if gforce detect a accident ( more mouvement not linear in all direction).

Can you add this 2 crash features?​

**Comment by dnewman-gpsw** [comment]
Use an accelerometer based IMU trigger. https://gopro.github.io/labs/control/imutrigger/

**Reply by viskor** [reply]
So ... what is your recommandation about the config imu? > https://www.youtube.com/watch?v=3_rY0PXre-o

**Reply by viskor** [reply]
I have make a New post for upgrade the imu trigger because its a good solution but not complete
